[2024-11-18T09:41:13.303+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T09:41:22.537+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.task_test manual__2024-11-18T09:40:07.011246+00:00 [queued]>
[2024-11-18T09:41:26.338+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.task_test manual__2024-11-18T09:40:07.011246+00:00 [queued]>
[2024-11-18T09:41:26.339+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-11-18T09:41:34.332+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task_test> on 2024-11-18 09:40:07.011246+00:00
[2024-11-18T09:41:37.221+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'task_test', 'manual__2024-11-18T09:40:07.011246+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpa98jc1y6']
[2024-11-18T09:41:37.490+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask task_test
[2024-11-18T09:41:37.509+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=651) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-18T09:41:37.566+0000] {standard_task_runner.py:72} INFO - Started process 672 to run task
[2024-11-18T09:41:50.114+0000] {job.py:229} INFO - Heartbeat recovered after 61.57 seconds
[2024-11-18T09:41:52.444+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_pipeline.task_test manual__2024-11-18T09:40:07.011246+00:00 [running]> on host 31a19111ad42
[2024-11-18T09:41:56.302+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='naphat' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='task_test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T09:40:07.011246+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T09:40:07.011246+00:00'
[2024-11-18T09:41:56.352+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T09:41:58.814+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-18T09:41:58.852+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is test for the first task is work!']
[2024-11-18T09:41:59.226+0000] {subprocess.py:99} INFO - Output:
[2024-11-18T09:41:59.560+0000] {subprocess.py:106} INFO - hello world, this is test for the first task is work!
[2024-11-18T09:41:59.586+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-18T09:42:00.527+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T09:42:00.552+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=task_test, run_id=manual__2024-11-18T09:40:07.011246+00:00, execution_date=20241118T094007, start_date=20241118T094122, end_date=20241118T094200
[2024-11-18T09:42:01.038+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T09:42:01.358+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-18T09:42:01.389+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
